File: src/main/java/fr/lewon/web/bot/entities/input/contest/ActionContestGiveReward.java
package fr.lewon.web.bot.entities.input.contest;

import fr.lewon.bot.http.body.urlencoded.FUEMember;

public class ActionContestGiveReward extends ActionContest {

	@FUEMember("id_contest")
	private Integer idContest;
	
	public ActionContestGiveReward(Integer idContest) {
		super("give_reward");
		this.idContest = idContest;
	}

}

File: src/main/kotlin/fr/lewon/bot/hh/entities/input/contest/ActionContestGiveReward.kt
package fr.lewon.bot.hh.entities.input.contest

import com.fasterxml.jackson.annotation.JsonProperty

class ActionContestGiveReward(idContest: Int) : ActionContest("give_reward") {

    @JsonProperty("id_contest")
    val idContest: Int = idContest

}